SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Utils/Constants.cs

```diff
@@ -17,5 +17,8 @@ public static class Constants
         public const int DefaultTriggerCount = 50;
         public static readonly TimeSpan DefaultTriggerInterval = TimeSpan.FromSeconds(3);
         public const int DefaultTriggerMaxParallelRequests = 3;
+
+        // File Storage
```

Comment:
We use dots at end of comments, please fix that in at least the new or modified comments in this PR.

Reply:
```suggestion
        // File Storage.
```